ursabot/builders.py

```diff
@@ -325,7 +326,19 @@ def builders_for(cls, workers, images=tuple(), **kwargs):
     generator='Ninja',
     definitions=definitions
 )
-cpp_compile = Ninja(name='Compile C++', workdir='cpp/build')
+
+
+@util.renderer
+def ninja_concurrency(props):
+    n_cpus = len(os.sched_getaffinitiy(0))
```

Comment:
This runs on the master side.

Reply:
Ok, I'll switch it to a `None` otherwise, and branch the argument accordingly.